src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -377,7 +377,7 @@ function canopy_sw_rt_two_stream(
 
     # Convert fractional absorption into absorption and return
     # Ensure floating point precision is correct (it may be different for PAR)
-    F_trans = (1 - F_abs - F_refl) / (1 - α_soil)
+    F_trans = (1 - F_abs - F_refl)
```

Comment:
This formula is correct as is when alpha_soil < 1.
When alpha_soil = 1, the formula is not correct, I think we should have F_trans = F_abs + F_refl -1

Let's see if we can rework this so the output is correct in either case